### DISCUSSION TITLE
Chat between operators

### DISCUSSION THREAD

**Discussion by akshay-xp** [LiveHelperChat/livehelperchat#1925] [Q&A]
Hello,
Is it possible for exactly two operators to chat, through LHC restapi only?
I tried to implement through `group chat` ([operators_chats](https://api.livehelperchat.com/#/operators_chats))  but i think it is lacking some features that normal chat already has. 

1. I can't find any mention of 'chat status' and 'unread messages' in api response
2. I can only send messages through `restapi user`, i am not able to override add message api with a `user_id`

Can I use normal chat to simulate chat between two operators? (using restapi only)

Thank you.

**Comment by remdex** [comment]
If it's not there means it's absent. You can take a look how the mobile app does that.
